src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/documents.html.twig

```diff
@@ -0,0 +1,111 @@
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>
+        {{ 'Date'|trans({}, 'Admin.Global') }}
+      </th>
+      <th>
+        {{ 'Document'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </th>
+      <th>
+        {{ 'Number'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </th>
+      <th>
+        {{ 'Amount'|trans({}, 'Admin.Global') }}
+      </th>
+      <th class="text-right">
+        {{ 'Actions'|trans({}, 'Admin.Global') }}
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for document in orderForViewing.documents.documents %}
+    <tr>
+      <td>
+        {{ document.createdAt|date('Y-m-d') }}
+      </td>
+      <td>
+        {% if document.type == 'invoice' %}
+          {{ 'Invoice'|trans({}, 'Admin.Global') }}
+        {% elseif document.type == 'credit_slip' %}
+          {{ 'Credit Slip'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {% else %}
+          {{ 'Delivery slip'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {% endif %}
+      </td>
+      <td>
+        {% if document.type == 'invoice' %}
+          <a class="_blank"
+             href="{{ getAdminLink('AdminPdf', true, {'submitAction': 'generateInvoicePDF', 'id_order_invoice': document.id}) }}"
+          >
+            {{ document.referenceNumber }}
+          </a>
+        {% elseif document.type == 'delivery_slip' %}
+          <a class="_blank"
+             href="{{ getAdminLink('AdminPdf', true, {'submitAction': 'generateDeliverySlipPDF', 'id_order_invoice': document.id}) }}"
+          >
+            {{ document.referenceNumber }}
+          </a>
+        {% elseif document.type == 'credit_slip' %}
+          <a href="{{ getAdminLink('AdminPdf', true, {'submitAction': 'generateOrderSlipPDF', 'id_order_slip': document.id}) }}">
+            {{ document.referenceNumber }}
+          </a>
+        {% endif %}
+      </td>
+      <td>
+        {% if document.amount %}
+          <p class="mb-0">{{ document.amount }}</p>
+
+          {% if document.amountMismatch %}
+            <p class="mb-0">{{ document.amountMismatch }}</p>
+          {% endif %}
+        {% else %}
+          --
+        {% endif %}
+      </td>
+      <td class="text-right">
+        {% if document.addPaymentAllowed %}
+          <a href="#">
+            {{ 'Enter payment'|trans({}, 'Admin.Orderscustomers.Feature') }}
+          </a>
+        {% endif %}
+
+        {% if document.type == 'invoice' %}
+          <a href="#">
+            {% if document.note is empty %}
+              {{ 'Add note'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
What would be the difference between this document note and the private note mentioned elsewhere on this page?

cc @MatShir